sessions/html-forms/zoo/css/styles.css

```diff
@@ -0,0 +1,61 @@
+*,
+*::before,
+*::after {
+	box-sizing: border-box;
+}
+
+body {
+	font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif,
+		'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
+	margin: 20px auto;
+	max-width: 300px;
+}
+
+form {
+	display: block;
+}
+
+input:not(input[type='checkbox']) {
+	width: 100%;
+	padding: 10px;
+	margin-bottom: 10px;
+	border: 1px solid #ccc;
+}
+
+.radio-options {
+	margin-bottom: 10px;
+	display: flex;
+	align-items: flex-start;
+}
+
+.radio-options > legend {
+	background-color: #c8f;
+	color: white;
+	padding: 0.2rem;
+}
+
+textarea {
```

Comment:
I would add a width to the textarea. This way the textarea fits better into the layout and the students don't have to deal with it further.